### DISCUSSION TITLE
Webmail subbdomain on Namecheap with Acme/LetsEncrypt - HOW?

### DISCUSSION THREAD

**Discussion by ewebgh33** [acmesh-official/acme.sh#5052] [Q&A]
Hi
Can anyone who uses acme with Namecheap help me figure this out?
I have certificates for domains working with acme.sh and LetsEncrypt.

But Namecheap adds a webmail / mail / cpanel subdomain, for every domain. 

I tried adding a certificate for the webmail subdomain. In terminal (via Cpanel) this seems be OK. eg:

```
acme.sh --issue -d webmail.domain.net -w /home/dir/domain.net
acme.sh --deploy --deploy-hook cpanel_uapi --domain webmail.domain.net

```
On deploy, it says 
`Successfully deployed certificate to 0 of 0 sites via UAPI`

Why? Why does it not deploy?
Also - I can't seem to add it manually, because in Namecheap's version of Cpanel, you can't install an SLL separately for a subdomain. 
This seems dumb, because if I create a subdomain, I have to re-issue the cert for EVERYTHING to cover 1 new subdomain?

ie
Do I instead, have to bundle the subdomains with the main domain.net and www.domain.net, into ONE certificate?
And each time I make a new subdomain (which might happen each week, for example client1.domain.net, client2.domain.net
I have to reissue?

DNS wildcard certs seems risky and problematic... docs mentioned doing this could wipe my records? Don't really want to run that chance.

**Comment by nmvega** [comment]
@ewebgh33
Did you ever figure this out? I'm facing the same issue. After successfully **issuing** an SSL Cert, **deploying** it with the same command results in:

```
Successfully deployed certificate to 0 of 0 sites via UAPI
```
